CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ Markdown Spec](https://github.github.com/gfm/).
 ### Contributors
 -->
 
+## [v1.0.5] - Fix configuration for IIIF server
+[v1.0.6]: https://github.com/open-oni/open-oni/compare/v1.0.5...v1.0.6
+
+### Changed
+- Pinned to RAIS 4 for local development and updated configuration for that version
```

Comment:
Thought I had this in the last "review", but can we remove "for local development" here since it was actually broken in AWS deployment too? We don't really dictate where this is deployed.

Also you can list yourself as a contributor ;)